Makefile

```diff
@@ -65,10 +65,11 @@ check-radoslib: ## Checks if radoslib is installed and if it contains the right
 	fi
 
 .PHONY: run-local-rookify
-run-local-rookify: ## Runs rookify in the local development environment (requires setup-venv)
-	$(eval PYTHONPATH="${PYTHONPATH}:$(pwd)/src")
-	source ./.venv/bin/activate && \
-	cd src && python -m rookify
+run-local-rookify: ## Runs rookify in the local development environment (requires setup-venv and currently requires build of the )
```

Comment:
Would it be nicer to have a dependency to target `setup-venv` instead of calling `setup-venv` separately here?